addons/point_of_sale/static/src/js/custom_hooks.js

```diff
@@ -161,5 +161,32 @@ odoo.define('point_of_sale.custom_hooks', function (require) {
         };
     }
 
-    return { useErrorHandlers, useAutoFocusToLast, onChangeOrder, useBarcodeReader, useAsyncLockedMethod };
+    function useCheckLotsIds() {
```

Comment:
Can you change your use of "lots ids" with "lot names". E.g. rename this method to `useCheckLotNames`.